wazo_service/main.py

```diff
@@ -79,25 +79,28 @@ def status(service_group: Iterable[Service]) -> int:
 
 
 def main() -> None:
-    parser = argparse.ArgumentParser(description='Process some integers.')
+    parser = argparse.ArgumentParser(description='Manage Wazo Services')
     parser.add_argument('action', help='Available actions: status')
     parser.add_argument(
         'service_group_name',
         default='default',
         nargs='?',
-        help='Available groups: all, default, xivo',
+        help='Filter by service group',
+        choices=list(SERVICE_GROUPS) + ['xivo'],
     )
     args = parser.parse_args()
-
-    service_group = SERVICE_GROUPS[args.service_group_name]
+    group_name = args.service_group_name
+    if group_name == 'xivo':
+        print('Warning: xivo is a deprecated alias to wazo: use wazo instead')
```

Comment:
This warning is redundant with the one in wazo-service
```
root@debian:~# wazo-service status xivo
Warning: xivo action is a deprecated alias to wazo: use wazo-service wazo
Warning: xivo is a deprecated alias to wazo: use wazo instead
Checking database...
```

Reply:
yep :laughing:

Reply:
Ok, I removed the warning, but changed the bash script to pass wazo instead of xivo. I also did that for wazo-nomonit and xivo-nomonit. They aren't documented, but would cause an error.